script/tool/lib/src/publish_check_command.dart

```diff
@@ -56,12 +119,21 @@ class PublishCheckCommand extends PluginCommand {
       final Colorize colorizedError = Colorize('$error\n$joinedFailedPackages')
         ..red();
       print(colorizedError);
-      throw ToolExit(1);
+    } else {
+      final Colorize passedMessage =
+          Colorize('All packages passed publish check!')..green();
```

Comment:
Looking at the new tests, it's very strange to have terminal color commands in the `--machine` output. I think we should replace all the Colorize calls with a utility method like:
```
_printImportantStatusMessage(String message, {@required bool isError}) {
  if (argResults[_machineFlag] as bool) {
    print('${isError ? 'ERROR' : 'SUCCESS'}: $message');
  } else {
    final Colorize colorizedMessage = Colorize(message);
    if (isError) {
      colorizedMessage.red();
    } else {
      colorizedMessage.green();
    }
    print(colorizedMessage);
  }
}

Reply:
I updated so that both machine mesasge and human message include the `error` and `success` prefix.